11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md

```diff
@@ -0,0 +1,265 @@
+    | Predefined allowed file types  | pdf, png, jpg, gif, txt  |
+
+36. Click **Submit**.
+37. Click the **Add Question** button to add the next question.
+38. Enter the following details:
+
+    | Field Name         | Value               |
+    |--------------------|---------------------|
+    | Enter question     | Are you a Robot?    |
+    | Choose answer type | reCAPTCHAv2         |
+    | Theme              | light               |
+    | Size               | normal              |
+    | Mandatory          | Yes                 |
+
+39. Click **Submit**.
+
+    ![Add questions](images/contact-us-form-add-questions.png)
+40. Click **Reorder**.
+41. Drag the **Data consent** group below the **Information** group.
+42. Click **I am done reordering**.
+43. Click **Save**.
+
+## Configure Workflow
+
+To configure the form workflow, follow these steps:
+
+1. Click on **Submit message/ Go to page**.
+2. Enter a customized message in the **Message on Submit** field.
+3. Click **Submit**.
+4. Click on **Send template email to xxx@xx.dk**.
+5. Select **Example-Template.cshtml** in the **Email template** field.
+6. Enable **Attachments**.
+7. Enter an email address in the **Sender Email** field.
+8. Click **Submit**.
+9. Click **Save**.
+
+## Configure Form Settings
+
+To configure the form settings, follow these steps:
+
+1. Select **Yes, keep submitted records in a database so they can be viewed and exported later** in the **Store Records** field.
+2. Enter the label for **Submit**, **Next**, and **Previous** buttons in the **Captions** field.
+3. Provide a stylesheet class in the **Styling** field.
+4. Enter the **Mandatory Error Message** and **Invalid Error Message** in the Validation field.
+5. Select **Mark Mandatory fields** in the **Mark fields**.
+6. Select the **Autocomplete** attribute value if you wish to use the autocompletion behavior.
+7. **Enable post moderation** if you wish the form submissions should be verified.
+8. Enable **display default fields** to select which fields are displayed in the list of form entries.
+9. Enable **Retain submitted records forever** and/or **Retain approved records forever** if you wish to retain data.
+10. Click **Save**.
+
+## Adding the Contact Us Form in the Content Node
+
+To add the Contact Us Form in the Content Node, follow these steps:
+
+1. Navigate to the **Content** section.
+2. Go to the **Contact Us** Page.
+3. Select **Insert macro** in the **Contact Form** field.
+4. Select **Insert Form with Theme**.
+5. Select **Add** in the **Choose a form** field.
+6. Select the **Contact Us** Form.
+7. Select **Add** in the **Theme** field.
+8. Select the **default** theme.
+
+    ![Adding the Contact Us Form](images/select-form.png)
+9. Click **Submit**.
+10. Click on **Save and Publish**.
+
+## Configure the reCAPTCHA value
+
+To configure the reCAPTCHA value, see the [Configuration](../../developer/configuration/README.md#recaptcha-v2-field-type-configuration) article.
```

Comment:
Should it be v3? 🤔

Reply:
No, I've used v2 in the video as v2 requires user to click I'm not a robot checkbox and select the image recognition challenge. v3 is more score based and used for feedbacks